src/lib/entity-state.ts

```diff
@@ -486,7 +485,13 @@ export abstract class EntityState<T> {
     return entities;
   }
 
-  private setup(storeClass: Type<EntityState<T>>, ...actions: string[]) {
+  private setup(storeClass: Type<EntityState<T>>, actions: string[]) {
+    const baseProto = Object.getPrototypeOf(storeClass.prototype);
+    const notPresent = actions.filter(action => !(action in baseProto));
+    notPresent.forEach(action => {
```

Comment:
use .some instead forEach

Reply:
Had that before but then I lose information about what action is actually missing

Reply:
I do not understand why you need to show mistakes for everyone

Reply:
You are right. If there are multiple not present only the first one will show anyways.
These alternatives have the same behaviour:

```typescript
if (notPresent.length > 0) {
  throw new NoMatchingActionHandlerError(notPresent[0]);
}

const _notPresent = actions.find(action => !(action in baseProto));
if (_notPresent !== undefined) {
  throw new NoMatchingActionHandlerError(_notPresent);
}
```